bindings/wasm/crate/src/lib.rs

```diff
@@ -20,7 +20,7 @@ lazy_static! {
 pub fn new_voodoo_instance() -> String {
     let mut instances = INSTANCE_MAP.lock().unwrap();
     let handle = (instances.len() as u64).to_string();
-    instances.insert(handle.clone(), Cell::new(VoodooInstance::new()));
+    instances.insert(handle.clone(), RefCell::new(VoodooInstance::new()));
```

Comment:
RefCell seems odd here: Though WASM is odd so it might be the best way. Naively I would have thought we could handle borrows at compile time.